src/Sarif.Viewer.VisualStudio/FixSuggestedAction.cs

```diff
@@ -1,70 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved. 
-// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
-
-using System;
```

Comment:
>u [](start = 0, length = 1)

Not really deleted. Moved, along with a dozen other files, into the `Fixes` subdirectory -- but so greatly changed that Git can't tell it's the same.